UIforETW/UIforETWDlg.h

```diff
@@ -211,6 +211,13 @@ class CUIforETWDlg : public CDialog
 	int initialHeight_ = 0;
 	int lastWidth_ = 0;
 	int lastHeight_ = 0;
+	const int minWidth_ = 870;

```

Comment:
Can these be set to zero? Then the initialWidth_ checks would handle the minimum. Having hard-coded minimums in the source seems undesirable because the minimum depends on system settings and will change after future dialog edits.

Reply:
Agreed.  Updated.